C++/4. Advanced-Operators.md

```diff
@@ -153,7 +151,7 @@ true
 true
 ```
 
-**Note: These type of operators are mostly not used like this, and are instead used in while loops, for loops, if statements, and in ternary expressions (all of which will be covered later).**
+**Note: These type of operators are mostly not used like this, and are instead used in while loops, for loops, if statements, and in the conditional operator (all of which will be covered later).**
```

Comment:
'These types' or 'This type'

'conditional operators' not 'conditional operator'

Reply:
Regarding the first part of what you said, it already says "these types."

Reply:
no it says 'These type'
![image](https://user-images.githubusercontent.com/64354499/114749047-248c3b00-9d70-11eb-978f-c75cdd5c0b9e.png)

Your addition the second - third word

Reply:
oh wait it sounds correct okay nothing.

Reply:
No, I misread it, it should say "these types," I'll fix it later when I fix everything else.

Reply:
Updated, can you check it out now?